src/sagemaker/tensorflow/legacy_training.rst

```diff
@@ -0,0 +1,487 @@
+Preparing Legacy Mode training script
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
should we change these headers now that they're not underneath other things? i.e. change this one to ==== and change the others appropriately